pkg/driver/statter.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package driver
+
+import (
+	"fmt"
+
+	"golang.org/x/sys/unix"
+)
+
+type Statter interface {
+	StatFS(path string) (int64, int64, int64, int64, int64, int64, error)
+	IsBlockDevice(string) (bool, error)
+}
+
+var _ Statter = realStatter{}
+
+type realStatter struct {
+}
+
+func NewStatter() realStatter {
+	return realStatter{}
+}
+
+// IsBlock checks if the given path is a block device
+func (realStatter) IsBlockDevice(fullPath string) (bool, error) {
+	var st unix.Stat_t
+	err := unix.Stat(fullPath, &st)
+	if err != nil {
+		return false, err
+	}
+
+	return (st.Mode & unix.S_IFMT) == unix.S_IFBLK, nil
```

Comment:
I'm going to assume this only works on unix?

Reply:
Yeah, right now it is only working for unix